looper/conductor.py

```diff
@@ -119,9 +120,46 @@ def write_sample_yaml_prj(namespaces):
     return {"sample": sample}
 
 
+def write_custom_template(namespaces):
+    """
+    Plugin: Populates a user-provided jinja template
+
+    Parameterize by providing pipeline.var_templates.custom_template
+    """
+
+    def load_template(pipeline):
+        with open(namespaces["pipeline"]["var_templates"]["custom_template"], "r") as f:
+            x = f.read()
+        t = jinja2.Template(x)
+        return t
+
+    err_msg = (
+        "Custom template plugin requires a template in var_templates.custom_template"
+    )
+    if not "var_templates" in namespaces["pipeline"].keys():
```

Comment:
I think `x not in y` reads better than `not x in y` and is generally recommended style.